Tests/js/models/assets/ez-locationmodel-tests.js

```diff
@@ -53,7 +92,32 @@ YUI.add('ez-locationmodel-tests', function (Y) {
         tearDown: function () {
             this.model.destroy();
             delete this.model;
-        }
+        },
+
+        "The content info should be instance of eZ.ContentInfo": function () {
+            var contentInfoStruct = this.loadResponse.Location.ContentInfo;
+
+            this.model.set('contentInfo', contentInfoStruct);
+            Y.Assert.isInstanceOf(
+                Y.eZ.ContentInfo,
+                this.model.get('contentInfo'),
+                'Content Info should be instance of eZ.ContentInfo'
+            );
+            Y.Assert.areEqual(
+                contentInfoStruct.Content.Name,
+                this.model.get('contentInfo').get('name'),
+                'Should instantiate content info with the name'
+            );
+        },
+
+        "The content info should be instance of eZ.ContentInfo when set to undefined": function () {
+            this.model.set('contentInfo', undefined);
+            Y.Assert.isInstanceOf(
+                Y.eZ.ContentInfo,
+                this.model.get('contentInfo'),
+                'ContentInfo should be instance of eZ.ContentInfo'
+            );
+        },
```

Comment:
a test where you check that the content info is correctly parsed is missing. Similar to https://github.com/ezsystems/PlatformUIBundle/blob/master/Tests/js/models/assets/ez-contentmodel-tests.js#L315